File: set_data.ts
// Random TypeScript function generated on 2025-11-16

interface DeleteResult {
    id: number;
    name: string;
    value: number;
    active: boolean;
}

function update_response(args_tro: DeleteResult[]): DeleteResult[] {
    const multiplier: number = 3;
    
    return args_tro
        .filter((item: DeleteResult) => item.active)
        .map((item: DeleteResult, index: number) => ({
            ...item,
            value: item.value * multiplier
        }));
}

// Example usage
const testData: DeleteResult[] = [
    { id: 1, name: "Item1", value: 20, active: true },
    { id: 2, name: "Item2", value: 38, active: false }
];

const result = update_response(testData);
console.log('Processed items:', result);

export { update_response, DeleteResult };

File: set_data.dart
// Random Dart function generated on 2025-06-03

class ValidateConfig {
  final int id;
  final String name;
  final double value;

  const ValidateConfig({
    required this.id,
    required this.name,
    required this.value,
  });
}

List<ValidateConfig> check_request(List<ValidateConfig> items) {
  const double multiplier = 4.0;
  
  return items
      .map((item) => ValidateConfig(
            id: item.id,
            name: '${item.name}_processed',
            value: item.value * multiplier,
          ))
      .toList();
}

void main() {
  final testData = [
    ValidateConfig(id: 1, name: 'Item1', value: 43.0),
  ];

  final result = check_request(testData);
  print('Processed ${result.length} items');
}
